README.md

```diff
@@ -1,67 +1,67 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Ghazanfar Portfolio
-
-> My software development portfolio.
-
-
-## Built With
-
-- HTML and CSS
-- No Frameworks
-- Visual Studio and Git
-- Linters
-
-
-
-## Getting Started
-
-**This project is part of Microverse first module course. This is used to build the personal portfolio to showcase the skills.**
-**To get the local copy of this clone following repository or download as zip:**
-**https://github.com/gak92/Ghazanfar_Portfolio.git**
-
-
-## Accessibility Features Added
-
-- Title added
-- Updated heading in orders
-- Used alt attribute in image tags
-- Used aria-label in input fields
-- Used tabindex
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: Ghazanfar Ali [@gak92](https://github.com/gak92)
-- LinkedIn: Ghazanfar Ali (https://www.linkedin.com/in/ghazanfar-ali-9a4998a/)
-
-👤 **Author2**
-- GitHub: [@githubhandle](https://github.com/ogagaoghene)
-
-## Links (Online Version)
-
-- https://gak92.github.io/Ghazanfar_Portfolio/
-
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- This project is part of Microverse study program.
-- Thanks to the Microverse team for the great curriculum.
-- Thanks to Figma Design Template
-- Thanks to Code Reviewers for the insightful feedback.
-- A great thanks to My coding partners, morning session team, and standup team Partners for their contributions and suggestions.
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Ghazanfar Portfolio
+
+> My software development portfolio.
+
+
+## Built With
+
+- HTML and CSS
+- No Frameworks
+- Visual Studio and Git
+- Linters
+
+
+
+## Getting Started
+
+**This project is part of Microverse first module course. This is used to build the personal portfolio to showcase the skills.**
+**To get the local copy of this clone following repository or download as zip:**
+**https://github.com/gak92/Ghazanfar_Portfolio.git**
+
+
+## Accessibility Features Added
+
+- Title added
+- Updated heading in orders
+- Used alt attribute in image tags
+- Used aria-label in input fields
+- Used tabindex
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: Ghazanfar Ali [@gak92](https://github.com/gak92)
+- LinkedIn: Ghazanfar Ali (https://www.linkedin.com/in/ghazanfar-ali-9a4998a/)
+
+👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/ogagaoghene)
+

```

Comment:
- [ ] Please include the details of your coding partner as Author 2 as this is a collaborative project 👍